analyzers/src/SonarAnalyzer.CSharp/Rules/MemberInitializerRedundant.cs

```diff
@@ -69,9 +66,15 @@ public sealed partial class MemberInitializerRedundant : SonarDiagnosticAnalyzer
 
         private void CheckInstanceMembers(SonarSyntaxNodeReportingContext c, TypeDeclarationSyntax declaration, IEnumerable<ISymbol> typeMembers)
         {
-            var typeInitializers = typeMembers.OfType<IMethodSymbol>().Where(x => x is { MethodKind: MethodKind.Constructor, IsStatic: false, IsImplicitlyDeclared: false }).ToList();
```

Comment:
`typeInitializers` was ill-named and I confused it with field initializers or property initializers or object initializers, but it was none of these. I renamed a couple of methods from typeInitializer to constructor.